File: PasswordValidator.py
attempt = 0
success = False

while True:
    passcode = input("PIN: ")
    attempt += 1

    if code == "4321":
        success = True
        break

    elif attempt == 1:
        print("Correct! It only took you one try!")
        break

    print("Wrong")

if success == True:
    print("Correct! It took you {attempt} attempts")

File: PasswordValidator.js
let attempt = 0;
let success = false;

while (true) {
    let passcode = prompt("PIN: ");
    attempt += 1;

    if (passcode === "4321") {
        success = true;
        break;
    } else if (attempt >= 3) {
        console.log("Too many incorrect attempts!");
        break;
    }

    console.log("Wrong");
}

if (success === true) {
    console.log(`Correct! It took you ${attempt} attempts`);
}
